docs/alerts/monitors/add-monitor.md

```diff
@@ -1,28 +1,53 @@
 ---
 id: add-monitor
-title: Add a Monitor
-description: This document provides instructions for creating a monitor.
+title: Create a Monitor
+description: Learn how to create a Sumo Logic monitor.
 ---
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
 This document shows you how to create a monitor.
 
+
+## Step 1: Open New Monitor window
+
+#### From your Monitors page
 1. Go to the Monitors page by clicking **Manage Data** > **Monitoring** > **Monitors**.
 1. Click on the **Add** button > **New Monitor** to add a new Monitor. The creation dialog box will appear.
-   * You can configure Sumo Logic Monitors using [Terraform modules](https://github.com/SumoLogic/terraform-sumologic-sumo-logic-monitor).
-1. Select a **Monitor Type**, either Logs or Metrics. <br/>  ![trigger conditions for monitor.png](/img/monitors/trigger-conditions-monitor.png)
-1. Select a **Detection Method**, either Static or Outlier. <br/> ![outlier detection method.png](/img/monitors/outlier-detection-method.png)
+
+#### From your Dashboard
+1. From a Dashboard, hover your mouse over a panel, click the kebab icon, then **Open in Log Search**.
+1. From your log search view, click the kebab icon in the upper right corner, then **Create a Monitor**.
+
+#### From your Log Search
+1. Click the kebab icon in the upper right corner, then **Create a Monitor**.
+
+## Step 2: Select Monitor type and Detection Method
+
+1. Select a **Monitor Type** (Logs, Metrics, or SLO). <br/><img src={useBaseUrl('img/monitors/trigger-conditions-monitor.png')} alt="icon" width="300"/>
+1. Select a **Detection Method** (Static or Outlier). <br/><img src={useBaseUrl('img/monitors/outlier-detection-method.png')} alt="icon" width="200"/>
    * **Static** allows you to set specific threshold conditions. Use this detection method when you are alerting on KPIs that have well defined and constant thresholds for what's good and bad. For example, infrastructure metrics like CPU utilization, and memory.
    * An **Outlier** is an unusual change or a spike in a time series of a key indicator. Use this detection method when you are alerting on KPIs that don't have well-defined constant thresholds for what's good and bad. You want the Monitor to automatically detect and alert on unusual changes or spikes on the alerting query. For example, application KPIs like page request, throughput, and latency.  
```

Comment:
**Outlier** lets you detect an unusual change or a spike in a time series of a key indicator.

(Make it match the style for static).